harmony_netcdf_to_zarr/convert.py

```diff
@@ -124,7 +132,8 @@ def mosaic_to_zarr(input_granules: List[str], zarr_store: Union[FSMap, str],
 
 
 def _output_worker(output_queue: Queue, shared_namespace: Namespace,
-                   aggregated_dimensions: Set[str], input_granules: List[str]):
+                   aggregated_dimensions: Set[str], dim_mapping: DimensionsMapping,
+                   variable_chunk_metadata: Dict = {}) -> None:
```

Comment:
I added the -> None typehint to places where I saw it.  I'm not 100% sure that's correct...

Reply:
Yeah - for any function without an explicit return, there is technically a `None` return type. (Fun fact - if you explicitly put `return None` at the end of a Python function `pycodestyle` or `pylint` will ding you for it)

Reply:
I will assume this is correct then.